### DISCUSSION TITLE
Show logs from last session

### DISCUSSION THREAD

**Discussion by aaronliu0130** [audacity/audacity#6312] [Ideas & Feature Requests]
I want to debug why #5793 is only happening to me, and I'd need logs from the previous session to do that.

**Comment by LWinterberg** [comment]
A debugger is much more useful for that. The log only shows you information which a developer thinks are relevant for a troubleshooting user to know. For Audacity, this often would be whether or not a file opened, whether FFmpeg is loaded and such. A debugger lets you halt and step through a program, so you can see exactly what bit of code is being executed at the moment and where it goes wrong, and also lets you read out what values certain variables have, and even what exactly the CPU is up to. 

Debuggers are usually found in IDEs such as Visual Studio, but even VSCode can use them: https://code.visualstudio.com/docs/cpp/cpp-debug

You will need to build Audacity from source for them to work: https://github.com/audacity/audacity/blob/master/BUILDING.md

**Reply by aaronliu0130** [reply]
Oh dear. Is there nothing else Audacity outputs somewhere?
